languages/ruby/exercises/concept/booleans/.meta/design.md

```diff
@@ -0,0 +1,45 @@
+## Goal
+
+The goal of this exercise is to teach the student the basics of the Concept of nil in Ruby.
+
+## Learning objectives
+
+- Know that `true` is a Ruby object used to represent logical `true`
+- Know that everything but `false` and `nil` is truthy
+- Know that `false` is a Ruby object used to represent logical `false`
+- Know that `false` is falsey
+- Know that `nil` is a Ruby object used to represent an "empty" or "default" value
+- Know that `nil` is falsey
+
+## Out of scope
+
+- `TrueClass`
+- `FalseClass`
+- `NilClass`
```

Comment:
How can this be out of scope if the class is what provides these values?

Perhaps replace with "Reference" instead of indicating that they are out of scope.

Reply:
Ya, I had left that comment myself, that I had added them as NilClass was in the previous iteration as was in the issue, but I think they also would be good discussion points in either the `hints.md` or the `after.md` document for students to review.